src/logic.js

```diff
@@ -32,6 +33,9 @@ export function fixDataFormat(data, rows = []) {
 
   // sorting data initially to prevent changes in data ordering
   if (rows.length > 0) {
+    if (sortFunc) {
+      return sortFunc(rows, formattedData);
```

Comment:
I'm not sure how you are implementing your `sortFunc`, but would it be better to take advantage of javascript's sort function and pass the `sortFunc` to that? Something like:
```js
return rows.reduceRight((acc, row) => {
  return acc.sort(sortFunc);
}, formattedData);
```

Reply:
In our case we're not sorting at all so we really don't want to incur the cost of a sort. I agree this would be better though. I can conditionally call acc.sort based on the presence of a sortFunc. I'll try to get this in today.